Adafruit_Feather_Sense/feather_sense_sensor_demo/feather_sense_sensor_demo.ino

```diff
@@ -41,6 +47,30 @@ void setup(void) {
   bmp280.begin();
   lis3mdl.begin_I2C();
   lsm6ds33.begin_I2C();
+  // check for readings from LSM6DS33
+  sensors_event_t accel;
+  sensors_event_t gyro;
+  sensors_event_t temp;
+  lsm6ds33.getEvent(&accel, &gyro, &temp);
+  accel_array[0] = accel.acceleration.x;
+  accel_array[1] = accel.acceleration.y;
+  accel_array[2] = accel.acceleration.z;
+  accel_array[3] = gyro.gyro.x;
+  accel_array[4] = gyro.gyro.y;
+  accel_array[5] = gyro.gyro.z;
+  // if all readings are empty, then new rev
+  for (int i =0; i < 5; i++) {
+    if (accel_array[i] != check_array[i]) {
+      new_rev = false;
+    }
+    else {
+      new_rev = true;
+    }
+  }
```

Comment:
This is going to flip-flop `new_rev` back and forth between `true` and `false` as you go through the `for` loop. I'm thinking that's not what you want -- shouldn't it be sticky in one direction or the other (probably in the `false` direction since you start with `true`?)?

Reply:
Yes, maybe set it true above the loop and any positive data check can then toggle it false

Reply:
good point, adding a break if a value triggers it